### PR TITLE
Various scripts to report and fix duplicate private uploads ids

### PR DESCRIPTION
**Pull request by darinkrauss**
@jh-bate These are the scripts I used Friday night to repair and validate all of the duplicate private upload id issues. I want to get these into master.

### CODE REVIEW

**Comment by jh-bate** [comment]
sorry - missed this but looking now

File: duplicate_private_upload_ids/assign_users_upload_ids.sh [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    ;;
+  staging)
+    MONGO_OPTIONS="${MONGO_OPTIONS:-} -ssl --quiet"
+    DEVICEDATA_DATABASE="data_staging"
+    ;;
+  devel)
+    MONGO_OPTIONS="${MONGO_OPTIONS:-} -ssl --quiet"
+    DEVICEDATA_DATABASE="data"
+    ;;
+  test)
```
Comment (jh-bate) [inline duplicate_private_upload_ids/assign_users_upload_ids.sh]: which env is "test"?

File: duplicate_private_upload_ids/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+These scripts report and fix duplicate private uploads ids.  
+
+Note: Backup the Mongo database before executing these
+
+In a terminal on the mongo master node for the specified environment, execute the following commands:
+
+```
+# Set ENVIRONMENT (prod|staging|devel|test|local)
```
Comment (jh-bate) [inline duplicate_private_upload_ids/README.md]: Which environment does test refer too?

Comment (darinkrauss) [inline duplicate_private_upload_ids/README.md]: It was the environment I had stood up for my testing.  Basically, it is an environment like prod, but does not have a custom SSL-enabled build of mongo.

**Comment by jh-bate** [comment]
LGTM - I know it worked at least :)

**Pull request merged**